Description: Adjust inclusion condition for python bindings to work on Debian
  Issue was the absence of a Boost_PYTHON_LIBRARY variable.
Author: Michael Hanke <mih@debian.org>
Last-Update: 2013-12-31


diff --git a/src/python-bindings/CMakeLists.txt b/src/python-bindings/CMakeLists.txt
index a804992..3c9e8a0 100644
--- a/src/python-bindings/CMakeLists.txt
+++ b/src/python-bindings/CMakeLists.txt
@@ -1,6 +1,5 @@
 
-
-if ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND Boost_PYTHON_LIBRARY AND NOT SOLARIS )
+if ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND (Boost_PYTHON_LIBRARY OR Boost_PYTHON_LIBRARY_DEBUG OR Boost_PYTHON_LIBRARY_RELEASE) AND NOT SOLARIS )
   configure_file (
 	"${PROJECT_SOURCE_DIR}/src/python-bindings/test_driver.in"
 	"${CMAKE_CURRENT_BINARY_DIR}/test_driver"
@@ -65,4 +64,5 @@ if ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND Boost_PYTHON_LIBRARY AND NOT
       add_test(python_htcondor_test test_driver tests/htcondor_tests.py TestConfig TestVersion)
     endif ( LINUX )
   endif (BUILD_TESTING)
-endif ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND Boost_PYTHON_LIBRARY AND NOT SOLARIS )
+  #endif ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND Boost_PYTHON_LIBRARY AND NOT SOLARIS )
+endif ( WITH_PYTHON_BINDINGS AND PYTHONLIBS_FOUND AND (Boost_PYTHON_LIBRARY OR Boost_PYTHON_LIBRARY_DEBUG OR Boost_PYTHON_LIBRARY_RELEASE) AND NOT SOLARIS )
